pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -472,9 +466,8 @@ static void _session_to_ep(const session_t *session, sock_udp_ep_t *ep)
 
 static void _timeout_callback(void *arg)
 {
-    msg_t timeout_msg = { .type = DTLS_EVENT_TIMEOUT };
-    sock_dtls_t *sock = arg;
-    mbox_try_put(&sock->mbox, &timeout_msg);
+    int *is_timed_out = (int *)arg;
+    *is_timed_out = 1;
```

Comment:
could be combined to this:
```suggestion
    *(int *)arg = 1;
```

Reply:
changed to suggestion